### ISSUE TITLE
[A11y] - Best practice: Target does not meet minimum size

### ISSUE THREAD

**Issue by VaniChinnam** [dequelabs/cauldron#1397]
### Current Behaviour 
Best practice: Target does not meet minimum 24 x 24 px size. 
Note: Raising it as best practice as there is enough spacing but not minimum size of 24 x 24 px. 

### Expected Behaviour 
`RULE :  Targets MUST have an area of at least 24 by 24 CSS pixels or are spaced apart so that a 24 pixel diameter circle centered on the target does not touch another target nor a 24 pixel diameter circle placed on the center of any other adjacent targets that are less than 24 by 24 pixels. Exceptions exist. HOW TO FIX:  Fix this issue by doing at least ONE of the following: 1. Ensure that the control has a target size of at least 24 x 24 CSS px. 2. Ensure that a 24 pixel diameter circle centered on the target does not touch another target nor a 24 pixel diameter circle placed on the center of any other adjacent targets that are less than 24 by 24 pixels `<br/><hr/>
### Impact 
 <p>

- [ ] Blocker
- [ ] Serious
- [ ] Critical
- [ ] Moderate
- [x] Minor

### Issue Details

<details><summary>View Issue 1658566 </summary> 
 <p>

_**Auditor ID**_ : [1658566](https://axeauditor.dequecloud.com/issue-global-search?searchKeyword=1658566&searchOn=id) 

_**description**_ : 

Element description 1: The following "Icon button" (23 * 23px) does not meet the minimum size 
-Secondary edit icon
-Primary edit icon
-Error edit icon
-Tooltip Placement delete icon
-Tooltip Variant More Information icon
-Offscreen Text delete user icon
-Large edit icon
Element location: In the "Icon button" screen 

Element description 2: Panel header button
Element location: In the "Panel" page 

Element description 3: Thin pagination button (23 * 23)
Element location: In the "Pagination" screen 

Element description 4: links does not meet the minimum size 
-I am a link  (72.56*22)
-I am a link that looks like a thin primary button (281.56 *23)
Element location: In the "Link" page 

Element description 5: Breadcrumb "one/two/three/four" (26.55*18)  does not meet the minimum size 
Element location: In  the "Breadcrumb" page 

Element description 6: "Link" and "tag"  (32.13 * 22)
Element location: In the "Button" screen" 

Element description 7:  Value and "Label: Value" (50.67 *21)
Element location: In the "Tag" screen 

Element description 8: Progress step 1 2 3 (22.39 * 22.39)
Element location: In the "Tooltip Tabstop"

_**Page Name**_ : Project Wide 

_**source**_ : `<button class="IconButton IconButton--secondary" tabindex="0" type="button" aria-labelledby="tooltip134"><span aria-hidden="true" class="Icon Icon--pencil"><svg overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"></svg></span></button>` 

_**Screenshots**_ : [Thin pagination button](https://axeauditor.dequecloud.com/api/v1/file/2972fa90-bf9c-11ee-8bf8-a3abca247919
Panel button: https://axeauditor.dequecloud.com/api/v1/file/fb10de2a-bf9a-11ee-ba82-d711c0dd0a41
Icon button: https://axeauditor.dequecloud.com/api/v1/file/fb10e30c-bf9a-11ee-870f-ffdcecd645b4) 

</p>
</details>

**Comment by scurker** [comment]
Some of these issues may be addressed by #1396, but will directly address each one:

> Element description 1: The following "Icon button" (23 * 23px) does not meet the minimum size
> Element description 2: Panel header button
> Element description 3: Thin pagination button (23 * 23)

Will be fixed by #1396 

> Element description 4: links does not meet the minimum size
> Element description 5: Breadcrumb "one/two/three/four" (26.55*18) does not meet the minimum size

I think these would typically be link in text blocks, so there's nothing we can do to directly address these.

> Element description 6: "Link" and "tag" (32.13 * 22)
> Element description 7: Value and "Label: Value" (50.67 *21)

Same thing as above regarding links, but button tag should absolutely be bumped up to the minimum size which will be addressed by #1396.

> Element description 8: Progress step 1 2 3 (22.39 * 22.39)

This is an example of how to implement TooltipTabstop and isn't intended to be clickable. Given this is just a documentation artifact, it's likely we would address this directly.

Given this is a best practice and most of the issues will be addressed by #1396, I'm closing this issue as with won't fix.
